client/client.go

```diff
@@ -1025,11 +1025,20 @@ func (c *Client) retryIntv(base time.Duration) time.Duration {
 // registerAndHeartbeat is a long lived goroutine used to register the client
 // and then start heartbeatng to the server.
 func (c *Client) registerAndHeartbeat() {
+	// Before registering capture the hashes of the Node. The hashes may be out
```

Comment:
"hashes of the Node" isn't very descriptive- maybe say "captures the hashes of the Node's attributes and metadata".

Reply:
Done